operations/rack_resiliency/Enable_Rack_Resiliency_on_a_Running_System.md

```diff
@@ -0,0 +1,293 @@
+For details on doing this during an install or upgrade to CSM 1.7, see
+[Enable Rack Resiliency During Install or Upgrade](Enable_Rack_Resiliency_During_Install_or_Upgrade.md).
+
+1. [Enable and customize](#1-enable-and-customize)
+1. [Run Ansible plays](#2-run-ansible-plays)
+1. [Deploy Helm chart](#3-deploy-helm-chart)
+1. [Restart critical services](#4-restart-critical-services)
+
+## 1. Enable and customize
+
+Follow these steps to enable (and optionally customize) Rack Resiliency.
+
+1. (`ncn-mw#`) Retrieve the `customizations.yaml` file.
+
+    ```bash
+    TMPDIR=$(mktemp -d -p ~) &&
+    kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' \
+        | base64 -d > "${TMPDIR}/customizations.yaml" \
+        && echo "${TMPDIR}/customizations.yaml"
+    ```
+
+    Example output:
+
+    ```text
+    /root/tmp.iM4FrDrJEJ/customizations.yaml
+    ```
+
+1. (`ncn-mw#`) Enable the feature in `customizations.yaml`.
+
+    ```bash
+    yq write -i "${TMPDIR}/customizations.yaml" \
+        'spec.kubernetes.services.rack-resiliency.enabled' "true"
+    ```
+
+1. (`ncn-mw#`) Optionally, set custom zone name prefixes.
+
+    See [Zone names](Zones.md#zone-names) for details
+    on reasons for doing this and restrictions on names. This is optional; prefixes are not
+    required. However, **prefixes cannot be changed, set, or removed later**.
+
+    1. Optionally, set a site-specific [Kubernetes zone](Zones.md#kubernetes-zones) prefix.
+
+        > In the following command, replace `k8s-prefix-string` with the desired Kubernetes zone prefix.
+
+        ```bash
+        yq write -i "${TMPDIR}/customizations.yaml" \
+            'spec.kubernetes.services.rack-resiliency.k8s_zone_prefix' "k8s-prefix-string"
+        ```
+
+    1. Optionally, set a site-specific [Ceph zone](Zones.md#ceph-zones) prefix.
+
+        > In the following command, replace `ceph-prefix-string` with the desired Ceph zone prefix.
+
+        ```bash
+        yq write -i "${TMPDIR}/customizations.yaml" \
+            'spec.kubernetes.services.rack-resiliency.ceph_zone_prefix' "ceph-prefix-string"
+        ```
+
+1. (`ncn-mw#`) Update the `site-init` secret in the Kubernetes cluster.
+
+    ```bash
+    kubectl delete secret -n loftsman site-init \
+        && kubectl create secret -n loftsman generic site-init \
+            --from-file="${TMPDIR}/customizations.yaml"
+    ```
+
+    Expected output:
+
+    ```text
+    secret/site-init created
+    ```
+
+1. (`ncn-mw#`) Confirm that the fields are set to the desired values.
+
+    ```bash
+    kubectl get secrets -n loftsman site-init \
+        -o jsonpath='{.data.customizations\.yaml}' \
+        | base64 -d | yq r - 'spec.kubernetes.services.rack-resiliency'
+    ```
+
+    Example output (in a case where only the Ceph zone prefix was set):
+
+    ```yaml
+    enabled: true
+    ceph_zone_prefix: my-ceph-prefix
+    ```
+
+## 2. Run Ansible plays
```

Comment:
Following the procedures in this section will only run the Ansible plays on one master NCN and one storage NCN, but they should be run on all of them. 

In addition, this procedure will cause those nodes to re-run the entire configuration, not just the RR layer. Even for a node that has previously been configured, this can take a while. 

I think you could address both of these issues, and simplify this procedure considerably, by instead doing something like we did for CASMPET-7628. For that ticket, [this script](https://github.com/Cray-HPE/docs-csm/blob/release%2F1.7/scripts%2Foperations%2Fconfiguration%2Frefresh_worker_iscsi_config.py) was created to force just the iSCSI playbook to re-run on worker nodes. You could do something similar here